### PR TITLE
Move creating version info

### PR DESCRIPTION
**Pull request by yw-furiosa**
Obtaining version information at the time of building the rust package causes problems when publishing. Therefore, the code for obtaining version information was moved to the python binding.

### CODE REVIEW

File: device-api/Cargo.toml [hunk 1]
```diff
@@ -55,9 +55,6 @@ tracing-subscriber = { version = "0.3.1", features = ["env-filter", "json"] }
 [dev-dependencies]
 eyre = "0.6"
```
Comment (sukyoungjeong-furiosa) [inline device-api/Cargo.toml:57]:
```suggestion
```

File: bindings/python/Cargo.toml [hunk 2]
```diff
@@ -13,3 +13,8 @@ furiosa-device = { path = "../../device-api" }
 pyo3 = "0.18.1"
 pyo3-asyncio = { version = "0.18", features = ["attributes", "tokio-runtime"] }
 tokio = { workspace = true }
+
+
+[build-dependencies]
+chrono = "0.4.24"
+git2 = "0.17.1"
```
Comment (sukyoungjeong-furiosa) [inline bindings/python/Cargo.toml:20]:
```suggestion
[build-dependencies]
chrono = "0.4.24"
git2 = "0.17.1"

```

File: device-api/Cargo.toml (hunk 1 above)
Comment (yw-furiosa) [reply device-api/Cargo.toml:57]: Reflect at [6001c8e](https://github.com/furiosa-ai/device-api/pull/86/commits/6001c8e09125e8994a234009ddc4725639940d4c)

File: bindings/python/Cargo.toml (hunk 2 above)
Comment (yw-furiosa) [reply bindings/python/Cargo.toml:19]: Reflect at [6001c8e](https://github.com/furiosa-ai/device-api/pull/86/commits/6001c8e09125e8994a234009ddc4725639940d4c)

**Pull request merged**